pkg/config/modules_test.go

```diff
@@ -15,28 +15,22 @@ const (
 	testBaseUsr  = "testdata/modules/usr/share"
 )
 
-func testSetModulePaths() (func(), error) {
-	oldXDG := os.Getenv("XDG_CONFIG_HOME")
-	oldEtc := moduleBaseEtc
-	oldUsr := moduleBaseUsr
+func testSetModulePaths() {
+	t := GinkgoT()
 
 	wd, err := os.Getwd()
-	if err != nil {
-		return nil, err
-	}
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
 
-	if err := os.Setenv("XDG_CONFIG_HOME", filepath.Join(wd, testBaseHome)); err != nil {
-		return nil, err
-	}
+	t.Setenv("XDG_CONFIG_HOME", filepath.Join(wd, testBaseHome))
 
+	oldEtc := moduleBaseEtc
+	oldUsr := moduleBaseUsr
 	moduleBaseEtc = filepath.Join(wd, testBaseEtc)
 	moduleBaseUsr = filepath.Join(wd, testBaseUsr)
-
-	return func() {
-		os.Setenv("XDG_CONFIG_HOME", oldXDG)
+	t.Cleanup(func() {
```

Comment:
if we use ginkgo tests it should likely just call DeferCleanup directly

Reply:
Done